app/src/reducers/coverPage.js

```diff
@@ -0,0 +1,11 @@
+const initialState = null;
+import { GET_COVER_PAGE_ENTRIES } from '../actions';
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_COVER_PAGE_ENTRIES:
+      return action.payload;
```

Comment:
A reducer **must** always return a copy of the original state with modifications. Example:
```javascript
switch (action.type) {
  case VESSEL_INIT:
    return Object.assign({}, state, action.payload);
    break;
}
```